src/lib/viewers/controls/color-picker/ColorPickerControl.tsx

```diff
@@ -18,27 +18,31 @@ export default function ColorPickerControl({
     ...rest
 }: Props): JSX.Element | null {
     const paletteRef = React.useRef<HTMLDivElement>(null);
+    const toggleRef = React.useRef<HTMLButtonElement>(null);
     const [isColorPickerToggled, setIsColorPickerToggled] = React.useState(false);
-    const [isPaletteActive, handlers] = useAttention();
+    const [handlers] = useAttention();
+    const { current: paletteEl } = paletteRef;
```

Comment:
Can we revert this change? We typically destructure these inside the event handler where they're used, since the value could theoretically change before they're invoked.